package.json

```diff
@@ -32,16 +32,16 @@
   "private": true,
   "dependencies": {
     "@balena/happy-eyeballs": "0.0.6",
-    "dbus": "^1.0.7",
+    "@balena/systemd": "^0.4.0",
     "got": "^12.5.3",
     "mdns-resolver": "^1.0.0",
     "semver": "^7.3.2",
     "sqlite3": "^5.0.11",
     "systeminformation": "^5.6.10"
   },
   "engines": {
-    "node": "^16.17.0",
-    "npm": "^8.15.0"
+    "node": ">=16.17.0",
```

Comment:
What do you think about this @Page- ? I want to use it as a forcing function to ensure updates to latest node happen faster.

Reply:
I don't think it'll help with that at all since the nodejs version used/tested will be decided by the dockerfile won't it?

Reply:
Yeah, you have a good point. I just don't want to restrict again the Node version. If this fails in Node 20, I want to hear about it

Reply:
Yeah it's fine to have like this, it's just unlikely to be a forcing function